File: src/main/java/com/redgrapefruit/mythicaltowers/common/tool/weapon/DoubleSwordItem.java
package com.redgrapefruit.mythicaltowers.common.tool.weapon;

import com.redgrapefruit.mythicaltowers.common.core.EffectConfig;
import net.minecraft.item.ToolMaterial;

import java.util.List;

/**
 * A double sword takes shorter to strike and hits harder
 */
public class DoubleSwordItem extends CustomSwordItem {
    public DoubleSwordItem(List<EffectConfig> configs, ToolMaterial material, int attackDamage, float attackSpeed) {
        super(configs, material, attackDamage, attackSpeed);
    }
}

File: src/main/kotlin/com/redgrapefruit/mythicaltowers/tool/weapon/DoubleSwordItem.kt
package com.redgrapefruit.mythicaltowers.tool.weapon

import com.redgrapefruit.mythicaltowers.core.EffectConfig
import net.minecraft.item.ToolMaterial

/**
 * A double sword takes shorter to strike and hits harder
 */
class DoubleSwordItem(configs: List<EffectConfig>, material: ToolMaterial?, attackDamage: Int, attackSpeed: Float) :
    CustomSwordItem(configs, material, attackDamage, attackSpeed)